src/barkhaPJun20/ExtractNumFromURL.java

```diff
@@ -0,0 +1,37 @@
+package barkhaPJun20;
+
+public class ExtractNumFromURL {
+	String output="";
+	
+	void displayItemId(String URL) {
+		System.out.println("Input is=> "+URL);
+		String removeFB=URL.replace("https://www.facebook.com/technocredits/", "");
+		if (removeFB.contains("itemId=")) {
+			if (removeFB.contains("itemId=") && !removeFB.contains("empid=123")) {
+				for (int index=0; index<removeFB.length(); index++) {
+					char ch=removeFB.charAt(index);
+					if (Character.isDigit(ch))
+						output+=ch;
+				}
+				System.out.println("itemID is=> "+output);
+			}else if(removeFB.contains("itemId=") && removeFB.contains("empid=123")) {  //will work as if
+				String removeEmpid=removeFB.replace("empid=123", "");
+				for (int index1=0; index1<removeEmpid.length(); index1++) {
+					char ch=removeEmpid.charAt(index1);
+					if (Character.isDigit(ch))
+						output+=ch;
```

Comment:
Your code is not optimize:
Try below solution-
if (input.contains("itemId=")) {
			String str = input.split("itemId=")[1];
			String output = "";
			for (int index = 0; index < str.length(); index++) {
				if (Character.isDigit(str.charAt(index))) {
					output += str.charAt(index);
				} else {
					break;
				}
			}
			System.out.println("Item id is " + output);
		} else {
			System.out.println("No itemID present.");
		}

Suggesting an alternative You can also write : 
try{
String str = input.split("itemId=")[1].replace("/", "").split("&")[0];
System.out.println("Item id is " + str);

}catch(StringIndexOutOfBoundsException siob){
			System.out.println("Item ID -> Not present");
		}

This two line in try block and it will work.